test/BPINN_Tests.jl

```diff
@@ -209,12 +212,12 @@ end
     @test mean(abs.(physsol1 .- meanscurve2_2)) < 5e-2
 
     # estimated parameters(lux chain)
-    param1 = mean(i[62] for i in fhsampleslux22[1000:1500])
+    param1 = mean(i[62] for i in fhsampleslux22[1000:length(fhsampleslux22)])
     @test abs(param1 - p) < abs(0.3 * p)
 
     #-------------------------- solve() call 
     # (lux chain)
-    @test mean(abs.(physsol2 .- sol3lux_pestim.ensemblesol[1])) < 0.15
+    @test mean(abs.(physsol2 .- pmean(sol3lux_pestim.ensemblesol[1]))) < 0.15
```

Comment:
`pmean` typo?

Reply:
nope, the mean is required as the solution's standard deviation are different at domain points, sometimes keeping this uncertainty is too large causing the tests to fail. so i just took the means for testing